src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -124,7 +126,14 @@ public CopyArtifact(String projectName, String parameters, BuildSelector selecto
 
         this.project = projectName;
         this.parameters = Util.fixEmptyAndTrim(parameters);
-        this.selector = selector;
+
+        if (selector == null) {
+            // TODO: specify the BuildSelector instance in the workflow script via CoreStep?
+            this.selector = new StatusBuildSelector(true);
```

Comment:
Use `@DataBoundSetter` to define a default value for a property. [Tips](https://github.com/jenkinsci/workflow-plugin/blob/master/scm-step/README.md#constructor-vs-setters)